.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Changes
+
+<!-- example:
+- Fix wrong color in button
+- Add a new page
+-->
+
+# Associated issue
+
+<!-- example:
+Resolves https://trello.com/c/oYgBIyqt/2-document-architecture-change-in-decision-log
+-->
+
+# How to test
+
+<!-- example:
+1. Open preview link: https://...
+2. Click on *x*
+3. Verify the button is red
+-->
+
+# Checklist
+
+<!-- Please strike through and check off all items that do not apply (rather than removing them) -->
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have made sure that my PR is easy to review (not too big, includes comments)
+- [ ] I have added/updated tests to prove that my feature works (if not possible please explain why)
+- [ ] I have made changes to the README if the change affects the architecture of the project (npm commands changed, new service added, environmental variable added)
+- [ ] I have notified a reviewer
```

Comment:
Thoughts on removing the checklist? I find them more annoying them helpful.

Reply:
I do like the checklist for more junior developers. I find it lowers the effort for reviewers (which are often the tech leads). How about a simplified one?

```md
- [ ] I have performed a self-review of my own work
- [ ] I have made sure that my PR is easy to review (not too big, includes comments)
- [ ] I have notified a reviewer
```

Reply:
Shorter is less of a chore, I just dislike the whole checklist, maybe if it had points which are unique to a project which I could forget I would use it. Either way we can merge it as is too if you prefer all checklist points. Don't want to block merging this.